src/App.js

```diff
@@ -41,15 +41,23 @@ class App extends Component {
 
   onItemClicked = () => {
```

Comment:
consider naming this something to convey it's meaning and saving `onItemClicked` for the event handler function in GameItem.js